src/ts/components/account/Progress.tsx

```diff
@@ -11,6 +15,17 @@ interface IProgressProps {
  */
 class Progress extends React.Component<IProgressProps> {
 
+  handleClick = route => event => {
```

Comment:
unused event generates a tslint warning here. perhaps we can replace it with `()`

Reply:
yeah, you are right